src/lib/DatePicker.svelte

```diff
@@ -252,10 +257,13 @@
           ><path d="M6 0l12 12-12 12z" transform="rotate(90, 12, 12)" /></svg
         >
       </div>
-      <div class="dropdown year">
+      <div class="dropdown year" tabindex="-1" bind:this={yearDiv}>
         <select
           value={browseYear}
-          on:input={(e) => setYear(parseInt(e.currentTarget.value))}
+          on:input={(e) => {
+            setYear(parseInt(e.currentTarget.value))
+            yearDiv.focus()
```

Comment:
Why do we focus the `monthDiv`/`yearDiv` and not `select`?

Reply:
You could do it to the select element itself, but since I don't like how the select elements have outlines when focused on, so I went for divs instead. 

Otherwise you could also choose to focus on the select elements and add extra css to remove those outlines.

Reply:
I wonder if the outline would show when using `:focus-visible`

Reply:
just a heads up that this PR had some failed tests and hasn't been merged still. personally having to maintain a fork, but would be nice if it's merged

Reply:
If you'd like, feel free to look into the comments and open a new PR @boblitex

Reply:
@probablykasper i guess we should not focus on select tag because here the problem is if we try to focus on select tag on input then also in IOS it is loosing focus, as i think its default behaviour of safari to loose focus once the option is selected.

so here we focus on wrapper div of select tag, this makes wrapper div as the related target in the onFocusOut event, and visible still remains true

```
function onFocusOut(e: FocusEvent) {
		if (
			e?.currentTarget instanceof HTMLElement &&
			e.relatedTarget &&
			e.relatedTarget instanceof Node &&
			e.currentTarget.contains(e.relatedTarget)
		) {
			return
		} else {
			visible = false
		}
	}
```

Reply:
if we give outline:none on focus for wrapper month and year divs , would that affect a11y?

Reply:
I mean for example screen readers, I don't think that would fix that